src/main/kotlin/no/nav/syfo/aktivitetskrav/api/AktivitetskravVurderingResponseDTO.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.syfo.aktivitetskrav.api
+
+import java.time.LocalDateTime
+
+data class AktivitetskravVurderingResponseDTO(
```

Comment:
Bruker `*ResponseDTO` for å lettere skille det fra DTOen som kommer i POST-endepunktet senere.